packages/runtime/container-runtime/src/summary/runningSummarizer.ts

```diff
@@ -842,12 +867,20 @@ export class RunningSummarizer extends TypedEventEmitter<ISummarizerEvents> impl
 		}
 
 		const { reason, ...summarizeOptions } = options;
-		const result = this.trySummarizeOnce(
-			{ summarizeReason: `onDemand/${reason}` },
-			summarizeOptions,
-			resultsBuilder,
-		);
-		return result;
+		if (options.retryOnFailure === true) {
+			this.summarizeOnDemandWithRetries(`onDemand;${reason}`, resultsBuilder).catch(
```

Comment:
This uses `;` but the else case uses `/`

Reply:
Yeah, I know. I tried changing it to `onDemand;` in the past for the other one but it started failing PR pipelines for some reason. I could not figure out why so I reversed that change. See https://github.com/microsoft/FluidFramework/pull/16886 for details.